plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -430,7 +430,8 @@ private class DokkaDescriptorVisitor(
         originalDescriptor: PropertyDescriptor,
         parent: DRIWithPlatformInfo
     ): DProperty {
-        val (dri, inheritedFrom) = originalDescriptor.createDRI()
+        val (dri, _) = originalDescriptor.createDRI()
+        val inheritedFrom = dri.copy(callable = null).takeIf { parent.dri.classNames != dri.classNames }
```

Comment:
Oh, it seems you've already answered that, but the pr comment was marked "outdated"

It would be really helpful to document some of this in code

> createDRI returns the DRI of the exact element and potential DRI of an element that is overriding it (It can be also FAKE_OVERRIDE which is in fact just inheritance of the symbol)
> 
> Looking at what PSIs do, they give the DRI of the element within the classnames where it is actually declared and inheritedFrom as the same DRI but truncated callable part, I think it is wise to unify that. Nonetheless, the second output of the tuple was never correct inheritedFrom value and has to be changed.
> 
> Now I think I need to double-check the dri if it is not a fake override, but an actual override. Also, I was thinking whether we could rely on existence overridenBy as the guard for inheritedFrom creation.

Reply:
Done